docs/modules/record-encryption/fortanix-dsm/con-fortanix-dsm-setup.adoc

```diff
@@ -0,0 +1,61 @@
+// file included in the following:
+//
+// assembly-hashicorp-fortanix-dsm.adoc
+
+[id='con-fortanix-dsm-setup-{context}']
+= Setup
+
+The filter integrates with the {fortanix-dsm}[Fortanix Data Security Manager (DSM)].  Both
+Fortanix DSM software-as-a service (SaaS) or an on-premise installation are supported.
+
+These instructions assume that you are using the
+{fortanix-support}/docs/clients-command-line-interface-cli-for-fortanix-data-security-manager[Fortanix DSM CLI,]
+but you may use the Fortanix DSM UI if preferred.
```

Comment:
```suggestion
These instructions assume that you are using the link:{fortanix-support}/docs/clients-command-line-interface-cli-for-fortanix-data-security-manager[Fortanix DSM CLI], but you can use the Fortanix DSM user interface if preferred.
```

Reply:
I wonder about adding lots of links to third-parties in terms of maintenance and whether it might be best to have a further reading in the overview with single links to each docs.

Reply:
Happy to be guided on best practice.  Personally, I find having the links to hand useful.  Finding the page you need on another's site is always hard.   Also from a support perspective, knowing the user has found and followed the right instructions is vital.    But I certainly take your point about link-rot.

Reply:
Something for consideration out of this PR.